finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,15 @@
+for (var i = 0; i <= 100; i++) {
+
+if (i % 3 === 0) {
+  console.log("Fizz");
+}
+else if (i % 5 === 0) {
+  console.log("Buzz");
+}
+else if (i % 3 === 0 && i % 5 === 0) {
```

Comment:
This elseif expression should come before the singular check for `i % 3` and `i % 5`. Did you try running this code?